### DISCUSSION TITLE
How to link an existing Android Studio Project to an existing repo?

### DISCUSSION THREAD

**Discussion by ys-git** [community/community#22257] [New to GitHub]
<p>Hey There,<br>
I’m relatively new to git so please help me out.<br>
A while ago I shared my Android Project on my GitHub and did some commits.</p>
<p>I had to reinstall Android Studio since my emulator would crash frequently.</p>
<p>The problem is I can see no option to link my Project again to that existing repo.VCS in Android Studio only seems gives an option to create a new repo and not to link with existing repo.<br>
Can anyone please guide me how to do it?</p>

**Comment by ys-git** [comment, marked as answer]
<p>Nevermind…figured it out myself.</p>

**Reply by alnori6** [reply]
help please :))

**Comment by nadiajoyce** [comment]
<p><a class="mention" href="/ys-git">@ys-git</a> That’s great! Would you mind sharing the solution you ended up using so that future users seeing this thread and having the same problem will know what to do?</p>

**Comment by ys-git** [comment]
<p>Sure.</p>
<p>What I did is cloned my repo to Android Studio and it lets me use the same repo to commit.</p>
<p>It was quiet simple and now I think how silly of me to ponder on such thing for days. </p>
<p>PS: Please see the screenshot.Hope it helps.</p>
<p><img src="https://user-images.githubusercontent.com/32202840/181069959-b9553664-b219-472c-8b4e-3472b626bd8b.png" alt="Screenshot_3.png" data-base62-sha1="y687rJha5eKXMT1PTvK03q2AONd" width="669" height="244"></p>

**Comment by barkhamisra** [comment]
<p>Please describe the solution as I am new to github and I am having the same problem as you had.</p>

**Comment by mpboom** [comment]
<p><a class="mention" href="/ys-git">@ys-git</a> Thanks so much for sharing the solution with the community!</p>
<p><a class="mention" href="/barkhamisra">@barkhamisra</a> Hi and welcome! Could you please be a bit more specific why the explaination of <a class="mention" href="/ys-git">@ys-git</a> is not helpful for you, for example by providing more details? If your problem is (a bit) different, please open up a new topic so we can assist you better (this topic is already marked as solved so won’t get a lot of attention, and also for organizational purposes it would be handy).</p>
